app-name/src/Entries.jsx

```diff
@@ -0,0 +1,34 @@
+import RenderEntry from './RenderEntry';
+import { data } from './data';
+
+export default function Entries({ onClick }) {
+  return (
+    <div data-view="entries" className="entries">
+      <div className="row">
+        <div className="column-full">
+          <div className="new-header">
+            <h2>Entries</h2>
+            <a
+              data-view="entry-form"
+              className="new-button"
+              id="entry-form-anchor"
+              onClick={() => onClick('create')}>
+              NEW
+            </a>
+          </div>
+        </div>
+      </div>
+      <div className="row">
+        {data.entries.length > 0 ? (
+          <ul className="entry-list">
+            <RenderEntry onClick={onClick} />
```

Comment:
I'm surprised that `onClick` can be used for both a click on the NEW button and a click on an entry. But, when you got to implementing Edit, you may have realized this and changed it.